sdk/web-pubsub/web-pubsub-client-protobuf/samples-dev/basicusage.ts

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * @summary Basic usage of web-pubsub-client
+ */
+
+import {
+  WebPubSubClient,
+  WebPubSubClientCredential,
+  SendToGroupOptions,
+  GetClientAccessUrlOptions,
+  WebPubSubClientOptions,
+} from "@azure/web-pubsub-client";
+import { WebPubSubProtobufReliableProtocol } from "@azure/web-pubsub-client-protobuf";
+import { WebPubSubServiceClient } from "@azure/web-pubsub";
+
+require("dotenv").config();
+
+const hubName = "sample_chat";
+const groupName = "testGroup";
+const serviceClient = new WebPubSubServiceClient(process.env.WPS_CONNECTION_STRING!, hubName);
```

Comment:
Is `WPS` a good abbreviation? WebPubSub? AWPS?

Reply:
That's what we used in previous js libraries